components/web/apps/org.wso2.carbon.apimgt.admin.app/src/main/fragments/policy-add/policy-add.hbs

```diff
@@ -140,8 +146,104 @@
             </select>
         </div>
     </div>
-</form>
+    {{#if @queryParams.subscription}}
+    <div class="page-header">
+        <h2><br></h2>
+        <p class="lead">{{i18n "burst.control.rate.limiting"}}</p>
+    </div>
+
+    <div class="form-group">
+        <label class="control-label col-sm-3">{{i18n "Request.Count"}} </label>
+        <div class="col-sm-6">
```

Comment:
This is not only using different Grid ratios in different devices. To work this perfectly I have come across that, even we use the same ratio over the devices we have to include all the classes with same value. Its better to include it.